Branch/BranchMainClass.m

```diff
@@ -209,11 +223,11 @@ - (Branch*) startWithConfiguration:(BranchConfiguration*)configuration {
 #endif
 
     // TODO: This is for debugging only.  Remove it.
-    [[NSNotificationCenter defaultCenter]
-        addObserver:self
-        selector:@selector(notificationObserver:)
-        name:nil
-        object:nil];
+//    [[NSNotificationCenter defaultCenter]
```

Comment:
I assume this is commented out, so it can be reused the next time we need to debug. If so, the direction "Remove it." could be replaced with something like "Stub it out before pushing to production". If the code won't be reused the next time we debug, we should should just delete it.

Reply:
I didn't have time to assess if this debug code would be worthwhile in the future.